versions/3.0.md

```diff
@@ -2287,25 +2283,32 @@ The following properties are taken directly from the JSON Schema definition and
 - minProperties
 - required
 - enum
-- type
-
-The following properties are taken from the JSON Schema definition but their definitions were adjusted to the OpenAPI Specification.
-Their definition is the same as the one from JSON Schema, only where the original definition references the JSON Schema definition, the [Schema Object](#schemaObject) definition is used instead.
-- items
-- allOf
-- oneOf
-- anyOf
-- not
-- properties
-- additionalProperties
+
+The following properties are taken from the JSON Schema definition but their definitions were adjusted to the OpenAPI Specification. 
+- type - Value MUST be a string. Multiple types via an array are not supported.
+- allOf - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
```

Comment:
@DavidBiesack I almost responded with a _"+1"_ to your comment but I think the _"Inline or referenced schema"_ as to being either a _Schema Object_ or a _JSON Reference_ to a _Schema Object_ defined elsewhere.  @webron's working seem fine to me.